tools/bismark/.shed.yml

```diff
@@ -1,18 +0,0 @@
-categories:
```

Comment:
We need such a file for every repository in the TS.
This file servers as metadata storage and can be used to upload tools to the TS from the commandline.